model/scratch/tests/test_slr.py

```diff
@@ -1,16 +1,12 @@
-import random
 import tempfile
-from typing import Type
 
-from dffml.repo import Repo, RepoData
-from dffml.model.model import ModelConfig
+from dffml_model_scratch.slr import SLR, SLRConfig
```

Comment:
Imports from plugin (`dffml_model_scratch`) should go after imports from `dffml`